core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -8829,6 +8829,120 @@ public void subTestIntervalSecondFailsValidation() {
     sql(sql2).ok(expected);
   }
 
+  @Test public void testQueryHint() {
+    final String sql = "select "
+        + "/*+ properties(k1='v1', k2='v2'), no_hash_join, Index(idx1, idx2) */ "
+        + "empno, ename, deptno from emps";
+    final String expected = "SELECT\n"
```

Comment:
What does `properties(k1='v1', k2='v2')` mean?

Reply:
It is a fake hint now, i just add it to have a test. For this demo hint, it means a hint with name `properties` and also this `properties` hint have options `k1=v1, k2=v2`